src/_gettsim/parameters/eink_st.yaml

```diff
@@ -276,3 +276,21 @@ rounding:
       base: 1
       direction: down
       reference: § 32a Abs. 1 S. 6 EStG
+  zu_verst_eink_y_tu:
```

Comment:
Can you add an explanation of what the "add" does here? Seems a bit weird to me. Also, annual income was only recorded in steps of 54 Euros? Not 100 DM / 1.95583 = 51.13 EUR ?

Reply:
Ah that was a typo, should have been 36, not 54. I didn't convert currencies, but just took the rules from the EStG/ the Website of the ministry of finance:

> Für die Einkommensteuerformel 2002/2003 ist zur Ermittlung des abgerundeten zu versteuernden Einkommens (zvE) das zu versteuernde Einkommen auf den nächsten, durch 36 € ohne Rest teilbaren €-Betrag abzurunden, wenn es nicht bereits durch 36 ohne Rest teilbar ist und um 18 € zu erhöhen.

Reply:
Makes sense for Euro-times. But the 54 before that?

Reply:
This is a rule in the EStG as well:

> Für Einkommensteuerformeln 1981 bis 2001 ist zur Ermittlung des abgerundeten zu versteuernden Einkommens (zvE) das zu versteuernde Einkommen auf den nächsten, durch 54 ohne Rest teilbaren DM-Betrag abzurunden, wenn es nicht bereits durch 54 ohne Rest teilbar ist. In 2001 ist dieser ermittelte Wert um 27 DM zu erhöhen.

Reply:
Oh, I am afraid you will have to convert that, then. We were thinking to have a possibility to specify unit=DM, but IIRC we never got around to that, yet?

Reply:
Ah, I thought we calculate everything in DM before 2002. 

I converted the DM values to EUR.

Reply:
That would be the goal eventually, but so far, we did not get around to it!